config/redirects.js

```diff
@@ -253,7 +253,7 @@ async function buildDotIoRedirects() {
     : await fetchGithubFile({
         owner: 'hashicorp',
         repo: 'consul',
-        path: 'website/redirects.next.js',
```

Comment:
This is an unrelated fix, builds fail without it. See https://github.com/hashicorp/consul/pull/12696 which renamed the file.

Reply:
Fixed in https://github.com/hashicorp/dev-portal/commit/010485344365b9c474dcb687975e05d0186dcdb7

Reply:
ah nice, i'll rebase and drop this 💖